appveyor.yml

```diff
@@ -1,14 +1,20 @@
 build: false
 environment:
   matrix:
-    - PYTHON_VERSION: 27
-    - PYTHON_VERSION: 33
-    - PYTHON_VERSION: 34
-    - PYTHON_VERSION: 35
-    - PYTHON_VERSION: 36
-platform:
-  - x86
-  - x64
+    - PLATFORM : x64
```

Comment:
technically correct, but a shorter way of expressing this (and smaller change) is just banning the three excluded envs (https://www.appveyor.com/docs/build-configuration/#exclude-configuration-from-the-matrix)

Reply:
I believe this was @kirit93's original approach. I'm not sure why he abandoned it.

Reply:
I thought it would be simpler to play around with the order of the jobs if I explicitly defined the matrix. But I can revert to the older approach that uses ```exclude``` since the order is just x64 followed by the two x86 jobs.